src/services/http.ts

```diff
@@ -174,6 +182,7 @@ const httpUtils = {
   del,
   get,
   post,
+  patch,
```

Comment:
Is the order here significant?  It seems a little odd (as in illogical) to have patch between post and postImage, though i assume the order doesn't matter except for readability.